bindings/go/configuration/extract/v1alpha1/config.go

```diff
@@ -0,0 +1,91 @@
+package v1alpha1
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	v1 "ocm.software/open-component-model/bindings/go/configuration/v1"
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+const (
+	// ConfigType defines the type identifier for transformation configurations
+	ConfigType = "extract.oci.artifact.ocm.software"
+)
+
+var scheme = runtime.NewScheme()
+
+func init() {
+	scheme.MustRegisterWithAlias(&Config{}, runtime.NewVersionedType(ConfigType, Version))
+}
+
+// Config represents the top-level configuration for the transformation.
+//
+// +k8s:deepcopy-gen:interfaces=ocm.software/open-component-model/bindings/go/runtime.Typed
+// +k8s:deepcopy-gen=true
+// +ocm:typegen=true
+type Config struct {
+	Type runtime.Type `json:"-"`
+	// TempFolder defines places where plugins and other functionalities can put ephemeral files under.
+	// If not defined, os.TempDir is used as a default.
+	TempFolder string `json:"tempFolder,omitempty"`
```

Comment:
This currently shares configuration settings with the filesystem config so this looks off to me.

Reply:
Ah, so maybe, this just contains the type?

Reply:
Added layer configuration. Please see if that makes sense. 🤔